frontend/tests/e2e-tests.sh

```diff
@@ -63,6 +72,15 @@ if [[ " ${SCRIPT_SHORT_ARGS[@]} " =~ " -h " ]] || [[ " ${SCRIPT_LONG_ARGS[@]} "
     exit 0
 fi
 
+ENV_CFG=$(find $VENV_PATH -name "pyvenv.cfg" -print -quit)
+if [[ ! -z $ENV_CFG ]]; then
+    VENV_PATH=$(dirname $ENV_CFG)
+    source $VENV_PATH/bin/activate
```

Comment:
We should `deactivate` to avoid contaminating the shell running the script

Reply:
I read that because it is a different shell that runs the script there is no risk of "contamination" and the venv is automatically deactivated just like all the exported env variables are lost after the script exit. I thought it would be simpler like this. But i could still make a condition in the cleanup to deactivate it if you think it is best.
Also what do you think about the problem with venv activation on a Windows OS ?